### DISCUSSION TITLE
What more is on the forge for the next release?

### DISCUSSION THREAD

**Discussion by joaocarvalhoopen** [ahrm/sioyek#186] [General]
Hello Ahrm

Currently I think that you already have a very complete PDF and EPUB Reader!

For Linux in my case Ubuntu 20.04, what I’m hopping for the next version is the Ubuntu bar Icon, and the functionality of portals. The capacity to see the nearest figure while also seeing the text is a really good and important feature for technical reading.

Then, the next step, it would be really good if one had updated documentation, because you have so many extremely good features in the program that aren’t documented and people like me are opening issues about new features and you normal response is it was already implemented and works like this :-)

Then make a new Release Build and make a cool video demoing all the main features of the Sioyek PDF and EPUB reader and then many more people will start using your software!

You are doing a incredible good work Ahrm,

The very best regards Ahrm,
João

**Comment by ahrm** [comment]
Thanks. The portal functionality should already be there in ubuntu releases (it was one of the first features of sioyek). Is it not working for you?

**Comment by joaocarvalhoopen** [comment]
Hello  Ahrm,

I tested again with the last version F12 - Portals,

I tested with this paper PDF: 
https://arxiv.org/pdf/2203.09584.pdf

I pressed F12 and a message appear in the center of the Portal “No portals yet”, and then I when with the arrow keys from the beginning to the end of the document and the portal never updated the figures. Then I clicked on the figures in the main windows, and the portal never updated the image.
That’s why I said that the I was waiting for the portals to be implemented.

Thank you,

The very best regards,
João

**Reply by ahrm** [reply]
This is the intended behaviour. Portals are not automatic (because there would be a lot of false positives). You have to create them manually. There are many ways to create portals:
* You can press `p` to mark the source and navigate to some other place and press `p` again to mark the destination
* You can  press `p` and click on a link to create a portal from the link to its destination
* You can press `p` and middle click to use smart jump to create a portal

**Comment by joaocarvalhoopen** [comment]
Thank you Ahrm for the explanation,

It works really well!

The very best regards,
João
